common/static/common/js/spec/discussion/view/response_comment_view_spec.js

```diff
@@ -164,10 +166,17 @@
                             }
                         };
                     });
+
+                    this.event = DiscussionSpecHelper.makeEventSpy();
+                    // All the way down in discussion/utils.js there's this line
+                    // element.after(...);
+                    // element is event.target in this case. This causes a JS exception, so we override the target
+                    this.event.target = $("body");
+                    $(this.event.target).prop("disabled", false);
```

Comment:
@bradenmacdonald unfortunately, no - it is not passed straight to utils.showLoadingIndicator - instead it is wrapped into jQuery object a [couple](https://github.com/edx/edx-platform/blob/4fd7f2936aefe9421c704c266d3f35bb3ca7f1a5/common/static/common/js/discussion/views/response_comment_view.js#L142) of [times](https://github.com/edx/edx-platform/blob/4fd7f2936aefe9421c704c266d3f35bb3ca7f1a5/common/static/common/js/discussion/utils.js#L205), so it ends up being a document again (not a spy).